UnityProject/Assets/Editor/AndroidGradleProjectModifier.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEditor.Android;
+
+public class AndroidGradleProjectModifier : AndroidProjectFilesModifier
```

Comment:
Please apply extra changes from https://github.com/Unity-Technologies/uaal-example/pull/104/files, it shows how to achieve the same without configuration manager